ui/src/submissions/conferences/components/ExistingConferencesAlert.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import PropTypes from 'prop-types';
+
+import { Button } from 'antd';
+import FieldInfoAlert from '../../common/components/FieldInfoAlert';
+import pluralizeUnlessSingle from '../../../common/utils';
+import ExistingConferencesDrawer from './ExistingConferencesDrawer';
+
+function ExistingConferencesAlert({
+  onDatesChange,
+  dates,
+  numberOfConferences,
+}) {
+  const [openingDate, closingDate] = dates;
+  const [isDrawerVisible, setDrawerVisible] = useState(false);
+
+  const showDrawer = useCallback(visible => setDrawerVisible(visible), []);
+
+  useEffect(
+    () => {
+      if (openingDate && closingDate) {
+        onDatesChange(openingDate, closingDate);
+      }
+    },
+    [onDatesChange, openingDate, closingDate]
+  );
+
+  return (
+    numberOfConferences > 0 && (
+      <>
+        <FieldInfoAlert
+          description={
+            <span>
+              <strong data-test-id="conferences-exist-alert-number">
+                {numberOfConferences}
+              </strong>{' '}
+              other {pluralizeUnlessSingle('conference', numberOfConferences)}{' '}
+              found in these dates.
+              <Button type="link" size="small" onClick={() => showDrawer(true)}>
+                Show more
+              </Button>
+            </span>
+          }
+        />
+        <ExistingConferencesDrawer
+          visible={isDrawerVisible}
+          onDrawerClose={() => showDrawer(false)}
```

Comment:
you have to set memoized callback (what is returned from `userCallback`) directly as a prop, without a wrapper arrow function.